yt_spoti_converter_django_application/views.py

```diff
@@ -1,5 +1,71 @@
 from django.shortcuts import render
+import os
+from django.shortcuts import redirect
+import json
+import requests
 
 
 def index(request):
-    return render(request, "index.html")
+    try:
+        # print("FOR DEBUG: ", os.environ.get("SPOTI_CLIENT_ID"))
```

Comment:
committing commented code doesn't help anyone. you can use a debug level logger if you see fit, but this seems to be just regular debugging for your code.